build/stf-run-ci/tasks/create_builds.yml

```diff
@@ -23,11 +23,25 @@
 
   - name: Kill first build since it will always fail (triggered on BuildConfig creation)
     ansible.builtin.shell: sleep 10 ; oc delete build {{ artifact.name }}-1 -n "{{ namespace }}"
+    ignore_errors: true
+    retries: 3
+    delay: 10
+    register: kill_build
+    until: kill_build.rc == 0
```

Comment:
This is added to cover the case where there's an error sending the request e.g. a timeout, which I saw a few times when testing this in zuul